src/resources/views/worklog/index.blade.php

```diff
@@ -57,3 +57,19 @@
     </div>
 
 @stop
+
+@section('scripts')
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/css/select2.min.css" rel="stylesheet" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/js/select2.min.js"></script>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/select2-bootstrap-theme/0.1.0-beta.10/select2-bootstrap.min.css" rel="stylesheet"/>
```

Comment:
It's so 2014 :)
We use webpack, so these assets need to be added with appshell. Please remove them from here and add to appshell's asset tree.